packages/broker/src/config/ConfigWizard.ts

```diff
@@ -153,7 +156,41 @@ const createPluginPrompts = (): Array<inquirer.Question | inquirer.ListQuestion
         default: true
     }
 
-    return [selectApiPluginsPrompt, ...portPrompts, minerPluginPrompt]
+    const wantToSetBeneficiaryAddressPrompt: inquirer.DistinctQuestion = {
+        type: 'confirm',
+        name: 'wantToSetBeneficiaryAddress',
+        // eslint-disable-next-line max-len
+        message: `By default the generated / imported Ethereum identity will be used for staking, would you like to change this to a different address (i.e. set a beneficiary address)?`,
```

Comment:
Could make the message shorter so that it is more understandable. E.g. we know from privateKeyAnswers whether we generated or imported the key, and choose message:
`By default the generated Ethereum identity...` or `By default the imported Ethereum identity`.

Reply:
A few more wordy suggestions,

`The Ethereum identity that has been generated or imported will be utilized for staking by default. Do you want to modify this and assign a different address as the beneficiary?`

Or if we know the generated/import decision,
`The Ethereum identity that has been generated will be utilized for staking by default. Do you want to modify this and assign a different address as the beneficiary?`

`The Ethereum identity that has been imported will be utilized for staking by default. Do you want to modify this and assign a different address as the beneficiary?`